app/Config/App.php

```diff
@@ -303,7 +303,7 @@ class App extends BaseConfig
 	 * (empty string) means no SameSite attribute will be set on cookies. If
 	 * set to `None`, `$cookieSecure` must also be set.
 	 *
-	 * @var string
+	 * @var 'Lax'|'None'|'Strict'
```

Comment:
```suggestion
	 * @var string
	 * @phpstan-var 'Lax'|'None'|'Strict'
```

Reply:
and for empty string?

Reply:
empty string is really not allowed either in the cookie spec or in the php code. we only use it to define that _no samesite_ will be set. But this behavior is now "obsolete" as MDN reports that modern browsers requires cookies to have `SameSite=Lax` if no samesite is given. This error is thrown in the console.